docs/content/en/docs/packages/packagesecrmigration.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Packages regional ECR migration"
+linkTitle: "Packages regional ECR migration"
+weight: 4
+description: >
+    Migrating packages ECR registry to packages registry for EKS Anywhere subscription
+---
+
+When an EKS Anywhere Enterprise Subscription is created, the AWS account that created the subscription is granted access to EKS Anywhere Curated Packages in the AWS Region where the subscription was created. If an existing EKS Anywhere cluster has curated packages access configured with the non-subscription based licens; we need to migrate the packages to consume the regional ECR registries.
+
```

Comment:
Can we add a high-level comment on the customer impact? Can this procedure cause downtime to packages?

Reply:
So it does cause the packages to roll out but its a helm update; so the new pods come up and the old ones get terminated. So ideally there should be no downtime. Will add a line for that.